asminc/cpu.mac

```diff
@@ -7,6 +7,9 @@ CPU_ISET_65C02     = $0010
 CPU_ISET_65816     = $0020
 CPU_ISET_SWEET16   = $0040
 CPU_ISET_HUC6280   = $0080
+CPU_ISET_M740      = $0100
+CPU_ISET_C39_EMUL  = $0200
+CPU_ISET_C39_NATIVE = $0400
 
 ; CPU capabilities
 CPU_NONE           = CPU_ISET_NONE
```

Comment:
You should add a `CPU_C39` name that combines the EMUL and NATIVE bits.  And maybe, it should be used in many of the library source files, instead of `CPU_ISET_C39_EMUL`.